omod/src/test/java/org/openmrs/module/fhir/resources/FHIRConditionResourceTest.java

```diff
@@ -0,0 +1,38 @@
+package org.openmrs.module.fhir.resources;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class FHIRConditionResourceTest {
+
+    @Before
+    public void setUp() throws Exception {
+    }
+
+    @After
+    public void tearDown() throws Exception {
+    }
+
+    @Test
+    public void getByUniqueId() {
```

Comment:
This test doesn't seem to do anything. Are the unit tests here To-Do / in progress?

Reply:
They are Todos

Reply:
Ah okay so we should hold off on reviewing/merging this PR until you finish the To-Do's?

Reply:
There is a lot of code being pushed of which I think some is ready for review. @corneliouzbett, please let us know if there's any code that you would like reviewed at this time.